### PR TITLE
feat(rns): refactors rns expiration date model + processing

### PR DESCRIPTION
**Pull request by jurajpiar**
according to the changes done in response to the issue #366 in [PR460](https://github.com/rsksmart/rif-marketplace-cache/pull/460).

### CODE REVIEW

File: src/migrations/scripts/1597921613888-initial.ts [hunk 1]
```diff
@@ -10,7 +10,6 @@ import { Sequelize as SequelizeTs } from 'sequelize-typescript'
  * createTable "rates", deps: []
  * createTable "rns_transfer", deps: []
  * createTable "rns_domain", deps: []
- * createTable "rns_domain_expiration", deps: [rns_domain]
```
Comment (julianmrodri) [inline src/migrations/scripts/1597921613888-initial.ts:13]: I think you would need to create a new migration script for the changes as these changes. @nduchak is this correct?

Comment (jurajpiar) [reply src/migrations/scripts/1597921613888-initial.ts:13]: done

File: src/services/rns/processor.ts [hunk 2]
```diff
@@ -375,14 +358,28 @@ function isValidEvent (value: string): value is keyof typeof commands {
...
-      await commands[eventData.event](logger, eventParser ? eventParser(eventData) : eventData, eth, services)
-    } else {
-      logger.error(`Unknown event ${eventData.event}`)
-    }
+const rnsProcessorFactory: RnsEventHandleFactory = (
+  logger,
+  eth,
+  services,
+  eventParser?
+) => async (eventData) => {
```
Comment (AuHau) [inline src/services/rns/processor.ts:370]: Please no arrow function, use named function instead.

File: tsconfig.json [hunk 3]
```diff
@@ -0,0 +1,5 @@
+{
```
Comment (AuHau) [inline tsconfig.json:1]: Don't commit `tsconfig.json`! If you need it, then symlink it from `tasegir` using `ln -s ./node_modules/tasegir/src/config/tsconfig.json ./tsconfig.json`. `experimentalDecorators` are present there...

**Review by AuHau — CHANGES_REQUESTED** [review]
Generally LGTM, but there are few things to tackle...

**Review by AuHau — APPROVED** [review]
LGTM! Now only fix CI pls!

**Review by nduchak — APPROVED** [review]
LGTM

**Pull request merged**